### PR TITLE
add ssh and dind

### PR DESCRIPTION
**Pull request by TAOXUY**

### CODE REVIEW

File: prow/prowjobs/GoogleCloudPlatform/esp-v2/esp-v2.yaml [hunk 1]
```diff
@@ -52,7 +52,9 @@ presubmits:
     always_run: true
     decorate: true
     labels:
+      preset-k8s-ssh: "true"
       preset-service-account: "true"
+      preset-dind-enabled: "true"
```
Comment (BenTheElder) [inline prow/prowjobs/GoogleCloudPlatform/esp-v2/esp-v2.yaml]: this doesn't match the title. are you actually using docker in docker? what for?

Comment (TAOXUY) [reply prow/prowjobs/GoogleCloudPlatform/esp-v2/esp-v2.yaml]: Just keep all the config with our gob prow config

Comment (TAOXUY) [reply prow/prowjobs/GoogleCloudPlatform/esp-v2/esp-v2.yaml]: yep, I think so.

**Comment by BenTheElder** [comment]
the new esp-v2 OWNERS should? be able to approve this now, I'll approve this one but please have your team review in the future.
/approve
/lgtm

**Comment by TAOXUY** [comment]
> the new esp-v2 OWNERS should? be able to approve this now, I'll approve this one but please have your team review in the future.
> /approve
> /lgtm

Hi Ben,

That is what I really what. But seems it doesn't work(our teammates' approval doesn't work).

**Pull request merged**